frontend/src/components/Field/YesNo/YesNo.tsx

```diff
@@ -44,6 +44,11 @@ export interface YesNoProps {
    * Color scheme of the component to render. Defaults to `primary`.
    */
   colorScheme?: FieldColorScheme
+
+  /**
+   * Whether the component is in a high contrast state.
```

Comment:
```suggestion
   * High contrast is used for better visibility of values when viewed in the disabled state.
```

Reply:
recommend to include why a setting exists in the doc instead of listing what it does, which should be self explanatory based on the variable name